### ISSUE TITLE
[FEAT] Remove/Relax Matplotlib version requirement

### ISSUE THREAD

**Issue by acarafat** [gem-pasteur/Integron_Finder#111]
Hello,

We are developing a genome annotation and visualization program `beav` that relies on `Integron Finder`. We are using Bioconda to distribute our program. Our visualization script requires `Matplotlib >= 3.5.2`, whereas the Integron Finder specifies the version ==3.3.3 in the Bioconda recipe. I was wondering if you can possibly remove/relax the requirement. 

Thank you so much!

**Comment by jeanrjc** [comment]
I guess it should be possible. Is there any pb with Matplotlib 3.5.2  @bneron ?

You could also propose a PR, we're short on time to dev Integronfinder right now.

**Comment by acarafat** [comment]
Thank you so much @jeanrjc. Should I go ahead to propose a PR in the Bioconda recipe?

**Comment by bneron** [comment]
yes originally I fixed the version to fix some problem in conda in package resolution. but IF works well with matplotlib 3.5.2
So if bioconda can solve the packages dependancies there is no reason to stick on 3.3.3 version

**Comment by acarafat** [comment]
@bneron @jeanrjc I relaxed the matplotlib version requirement from ==3.3.3 to >=3.3.3 in the Bioconda recipie yaml file. I'll test once the build complete. Thanks!

**Comment by acarafat** [comment]
@bneron @jeanrjc The bioconda build was successful: https://github.com/bioconda/bioconda-recipes/pull/43671
